app/dao/fact_notification_status_dao.py

```diff
@@ -154,23 +154,25 @@ def fetch_notification_status_for_service_for_today_and_7_previous_days(service_
         FactNotificationStatus.bst_date >= start_date,
         FactNotificationStatus.key_type != KEY_TYPE_TEST
     )
+    if str(service_id) in (current_app.config['HIGH_VOLUME_SERVICE']):
+        all_stats_table = stats_for_7_days.subquery()
```

Comment:
Optional: Put a comment in here to say why we are doing this with a TODO saying that this is a temporary fix